skiko/src/commonMain/kotlin/org/jetbrains/skiko/SkikoView.kt

```diff
@@ -6,7 +6,10 @@ interface SkikoView {
     // Input
     fun onKeyboardEvent(event: SkikoKeyboardEvent) = Unit
     fun onPointerEvent(event: SkikoPointerEvent) = Unit
+
+    @Deprecated("This method will be removed. Use override val input:PlatformInput")
```

Comment:
Is this message outdated? Does it mean `Use override val input: SkikoInput` ?

Reply:
I provide this warning to users of skiko. I they update skiko to the next version, they will see this warning.
Maybe it's unnecessary, because skiko still in pre-released versions, and we can break compatibility without warnings?

Reply:
Do I understand correctly that SkikoInputEvent is not applicable for ios anymore (it's not used for ios)?

Reply:
Yes, for iOS it's redundant and not used anymore

Reply:
Do IPADs support keyboards? Can we connect a keyboard to an IPAD?
If it's possible, does it mean that we need to add more methods to SkikoInput for ios? To handle not only text inputs, but keyboard events such as tab/arrows navigation?

Reply:
Yeah iPad support hardware keyboard. Our compose uikit samples already works with iPadOS.
iPadOS use same .ipa app packages like iOS.
We can add more methods to handle keyboard support right in actual SkikoInput interface.
This interface is supposed to be extensible

Reply:
And actually i want to add more methods in SkikoInput later to handle RTL text layout's

Reply:
Got it! thank you :)